tools/cp-caps/README.md

```diff
@@ -54,21 +54,22 @@ Show help info.
 
 ```bash
 $ python3 ./tools/cp-caps/rcp_caps_test.py -h
-usage: rcp_caps_test.py [-h] [-c] [-l] [-d] [-f] [-p] [-t] [-T] [-v] [-D]
+usage: rcp_caps_test.py [-h] [-v] [-D] [--test-all] [--test-csl] [--test-data-poll] [--test-diag-commands] [--test-frame-formats] [--test-link-metrics] [--test-tx-info] [--test-throughput]
```

Comment:
Is it considered to split test suites into separate files, so that we can filter them by `-p`?

https://docs.python.org/3/library/unittest.html#cmdoption-unittest-discover-p

Reply:
I think using `-k` option should be good enough?

https://docs.python.org/3/library/unittest.html#cmdoption-unittest-k

Reply:
I have updated the code to directly use the unittest methods to run all test cases. Both options `-p` and `-k` are supported.